Bottom: 47cf9c6c27f98e001bd577fa6a5e1cb258c79a69
Top:    bf8cb20b0854ae7447b69ae4a138b5166f306d41
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-05 18:44:23 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
